src/main/java/com/assure/movie/model/domain/Actor.java

```diff
@@ -11,7 +11,7 @@
 public class Actor implements Identifiable<Long> {
```

Comment:
The JPA specification mentions that an Entity should be implement Seriablizable interface, but in a specific scenario.
What is that senario? and taking into account that, Do these Entity classes need to implement Serializable?

Reply:
JPA suggest that the entity should be serializable if those entities are directly exposed to the presentation layer, because those entities can be stored in the httpd session for caching purposes, in my case it's not necesary to be implement as a serializable, because I am using the DTOs classes.

Reply:
Perfect! thanks